### PR TITLE
Trials feature branch

### PR DESCRIPTION
**Pull request by 187jjay187**
created tribute page as a trials project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <title>Tribute Page</title>
+  </head>
+  <body>
+    <main id="main">
+      <div id="title">
+        <h1>Dr. Norman Borlaug</h1>
+      </div>
+      <div id="img-div">
+        <img src="" alt="" id="image" />
+        <figcaption id="img-caption">
```
Comment (187jjay187) [inline index.html:17]: i suggest you add an image link/path to your src to fix the broken img line in your css

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+img {
+  max-width: 100%;
+  display: block;
+  height: auto;
+  margin: 0 auto;
+}
```
Comment (187jjay187) [inline styles.css:6]: i suggest adding an image to to your index files src to fix the broken image link/path

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+img {
```
Comment (187jjay187) [inline styles.css:1]: i suggest you double check that all your tag line names are correct in your styles.css as it is in your index.html file so the two files can work with each other to produce a result

Comment (187jjay187) [inline styles.css:1]: i suggest you double check that all your tag line names are correct in your styles.css as it is in your index.html file so the two files can work with each other to produce a result

File: styles.css (hunk 2 above)
Comment (187jjay187) [inline styles.css:6]: i suggest adding an image to to your index files src to fix the broken image link/path

File: index.html (hunk 1 above)
Comment (187jjay187) [inline index.html:17]: i suggest you add an image link/path to your src to fix the broken img line in your css

**Review by 187jjay187 — COMMENTED** [review]
# Required Changes

Please check my comments below and make the required changes

and please resubmit for a re-review

Thanks

**Review by mckent05 — APPROVED** [review]
Hi @187jjay187, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well-done! 🎉 

## Highlights
- Good job being polite with your review 👍 👍 
- Good job stating reasons for suggesting a change 👏 👏 

## Optional suggestions
**OPTIONAL** - I'd suggest you use emojis while giving reviews, this will help you sound even more friendly 😄 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**